vault/identity_store_util.go

```diff
@@ -1092,6 +1086,12 @@ func (i *IdentityStore) UpsertGroupInTxn(txn *memdb.Txn, group *identity.Group,
 	// Increment the modify index of the group
 	group.ModifyIndex++
 
+	if group.Alias != nil {
```

Comment:
This block is also present in https://github.com/hashicorp/vault/blob/master/vault/identity_store.go#L222 which needs to be removed.